starlite/config/app.py

```diff
@@ -161,6 +161,12 @@ class Config(BaseConfig):
     """
     Defaults to [DEFAULT_OPENAPI_CONFIG][starlite.app.DEFAULT_OPENAPI_CONFIG]
     """
+    opt: Optional[Dict[str, Any]]
```

Comment:
Related to previous:

```suggestion
    opt: Dict[str, Any]
```

Makes it easier to use the config object in plugins if it can be done.

Reply:
Ah, TIL why config has been created this way. Thanks for explaining